### DISCUSSION TITLE
Adding a new device, Polhemus Patriot, to PlusToolkit

### DISCUSSION THREAD

**Discussion by SaraVv** [PlusToolkit/PlusLib#903] [Q&A] [answered]
Hello, 

I want to use PlusToolkit with a new device, with Polhemus Patriot. I've seen that it doesn't appear among the devices that accept your software, so I'm following this guide to adapt it: (https://plustoolkit.github.io/devicecode).

I'm stuck creating the .h and .cxx files in PlusLib>\src\PlusDataCollection and I was wondering if you could give me a hand with it. I've tried to follow the recommendations and rely on other existing devices but I don't have enough knowledge of c++.

Thanks in advance for your help.

**Comment by adamrankin** [comment, marked as answer]
Unfortunately you'll have to find a developer with access to a Patriot and time available to help. Do you have access to any developers in your company/research group?

**Reply by SaraVv** [reply]
Thank you for your quick response. 
There are no developers in my research group, but we do have access to a Patriot and are in contact with the company.

**Reply by adamrankin** [reply]
Sadly there's not much help I can offer in terms of development support. If you are able to find a developer, we can help them ramp up quickly to create the device for you.

**Comment by anaghamg** [comment]
Hello, 
I am using polhemus as well. Can you give an update on your progress with integrating Polhemus with PLUS. If you could reach out to me at anagha.m@cdac.in that would be wonderful. Thanks.
